enlighten/_basecounter.py

```diff
@@ -180,6 +180,7 @@ def __init__(self, keywords=None, **kwargs):
         self.min_delta = kwargs.pop('min_delta', 0.1)
         self._pinned = False
         self.last_update = self.start = self._count_updated = time.time()
+        self._closed_time = None
```

Comment:
I don't like that the type changes here. If it's going to be a float, it should probably start with the value `0.0`.